sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -770,6 +770,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(Utils.isTesting)
 
+  lazy val ALLOW_READING_UNKNOWN_COLLATIONS =
```

Comment:
Just wonder why you made it `lazy`?

Reply:
I do not have strong reasoning why it is made `lazy`, I will change it to standard `val`.